src/projects.h

```diff
@@ -165,168 +167,320 @@ typedef struct { double u, v; } projUV;
+    double  e2s;                       /* second eccentricity squared */
+    double  e3;                        /* third  eccentricity */
+    double  e3s;                       /* third  eccentricity squared */
+    double  one_es;                    /* 1 - e^2 */
+    double  rone_es;                   /* 1/one_es */
+
+    /* The flattenings */
+
+    double  f;                         /* first  flattening */
+    double  f2;                        /* second flattening */
+    double  n;                         /* third  flattening */
+    double  rf;                        /* 1/f  */
+    double  rf2;                       /* 1/f2 */
+    double  rn;                        /* 1/n  */
+
+    /* This one's for GRS80 */
+    double  J;                         /* "Dynamic form factor" */
+
+    double  es_orig, a_orig;           /* es and a before any +proj related adjustment */
+
+
+
+    /*************************************************************************************
+
+                          C O O R D I N A T E   H A N D L I N G
+
+    **************************************************************************************/
+
+    int  over;                      /* Over-range flag */
+    int  geoc;                      /* Geocentric latitude flag */
+    int  is_latlong;                /* proj=latlong ... not really a projection at all */
+    int  is_geocent;                /* proj=geocent ... not really a projection at all */
+
+    enum pj_io_units left;          /* Flags for input/output coordinate types */
+    enum pj_io_units right;
+
+
+    /*************************************************************************************
+
+                       C A R T O G R A P H I C       O F F S E T S
+
+    **************************************************************************************/
+
+    double  lam0, phi0;                /* central longitude, latitude */
+    double  x0, y0;                    /* false easting and northing */
+
+
+
+    /*************************************************************************************
+
+                                    S C A L I N G
+
+    **************************************************************************************/
+
+    double  k0;                        /* General scaling factor - e.g. the 0.9996 of UTM */
+    double  to_meter, fr_meter;        /* Plane coordinate scaling. Internal unit [m] */
+    double  vto_meter, vfr_meter;      /* Vertical scaling. Internal unit [m] */
+
+
+
+    /*************************************************************************************
+
+                  D A T U M S   A N D   H E I G H T   S Y S T E M S
+
+    **************************************************************************************/
+
+    int     datum_type;                /* PJD_UNKNOWN/3PARAM/7PARAM/GRIDSHIFT/WGS84 */
+    double  datum_params[7];           /* Parameters for 3PARAM and 7PARAM */
+    struct _pj_gi **gridlist;          /* Description needed */
```

Comment:
"Description needed" is not a very useful comment. It would be good to have those descriptions made before merging. Or at least a ticket so it is not forgotten once this PR is merged.